css/logo-nav.css

```diff
@@ -181,22 +172,29 @@ a {
     padding-top: 10%;
     padding-bottom: 10%;
 }
-.notebook-features .share-set h2, .notebook-features .language-set h2{
-    font-size: 21px;
-    line-height: 1.5;
-    color: #766A6A;
-    padding-top: 5px;
-}
-.notebook-features .data-set h2, .notebook-features .widget-set  h2{
+.notebook-features h2{
     font-size: 21px;
     line-height: 1.5;
     color:#766A6A;
 }
```

Comment:
Merged the 2 above rules and simplified. I remove the 5pixel offset on 2 of these. There is still a bug at **exactly** 992 pixel width. See device inspector: 
![screen shot 2015-10-21 at 11 59 40](https://cloud.githubusercontent.com/assets/335567/10647074/4404243a-77eb-11e5-9f90-4c4b4233b7ed.png)

Do you see the gap between the 2 green bars ?

You probably have one need 500, the two small white line in the blue bar indicate 2 close css media-transition separated by 2-3 pixels, and one a 750-770 where you have a double white limit that look like a thick one.